### ISSUE TITLE
Can i read the features and labels in model_fn

### ISSUE THREAD

**Issue by 520jefferson** [nlpming/tf-estimator#1]
I want to distill the pytorch big model to tf small model, so the smalll model read the features  and the big model can call the features and return the logits. 
So i want to add the distillation in the model_fn in small model, but the features cannot be readed. 
what should i do?
